packages/components/addon/components/hds/table/th-sort.hbs

```diff
@@ -2,7 +2,13 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
-<th class={{this.classNames}} aria-sort={{this.ariaSort}} {{style width=@width}} ...attributes scope="col">
+<th
+  class={{this.classNames}}
+  aria-sort={{this.ariaSort}}
+  {{style width=@width minWidth=@width}}
```

Comment:
FYI from my experiments I've seen that we can't set just the `min-width` or in some cases the table will not respect the desired width, we have to declare both

Reply:
I would believe this just because of the other weird CSS things that apply to table elements. 😞

Reply:
@MelSumner can you expand? I don't understand the comment